src/overlays/actors/ovl_En_Dodojr/z_en_dodojr.c

```diff
@@ -246,148 +246,148 @@ s32 func_809F6CA4(EnDodojr* this, PlayState* play) {
         yDist = bomb->world.pos.y - this->actor.world.pos.y;
         zDist = bomb->world.pos.z - this->actor.world.pos.z;
 
-        if ((fabsf(xDist) >= fabsf(unkVec.x)) || (fabsf(yDist) >= fabsf(unkVec.y)) ||
-            (fabsf(zDist) >= fabsf(unkVec.z))) {
+        if ((fabsf(xDist) >= fabsf(maxBombRange.x)) || (fabsf(yDist) >= fabsf(maxBombRange.y)) ||
+            (fabsf(zDist) >= fabsf(maxBombRange.z))) {
             bomb = bomb->next;
             continue;
         }
 
         this->bomb = bomb;
-        unkVec = bomb->world.pos;
-        retVar = 1;
+        maxBombRange = bomb->world.pos;
+        foundBomb = true;
         bomb = bomb->next;
     }
 
-    return retVar;
+    return foundBomb;
 }
 
-s32 func_809F6DD0(EnDodojr* this) {
+s32 EnDodojr_CanEatBomb(EnDodojr* this) {
```

Comment:
Fwiw I agree with this and I think it should be named differently since it has a side effect

I don't have a better suggestion than `TryEatBomb` which would probably work

Reply:
Ok going with `TryEatBomb`, in 485e09c, thanks!